src/test/java/com/powsybl/openloadflow/ac/NonImpedantBranchDisablingTest.java

```diff
@@ -78,4 +87,41 @@ void testOpenBranch() {
         assertEquals(-600.0, network.getLine("L2").getTerminal2().getP(), LoadFlowAssert.DELTA_POWER);
         assertEquals(Double.NaN, network.getLine("L1").getTerminal2().getP(), LoadFlowAssert.DELTA_POWER);
     }
+
+    @Test
+    void testDisabledNonImpedantBranch() {
+        Network network = NodeBreakerNetworkFactory.create3Bars();
+        Switch c1 = network.getSwitch("C1");
+        c1.setOpen(true);
+
+        LoadFlowParameters parameters = new LoadFlowParameters();
+        OpenLoadFlowParameters olfParameters = OpenLoadFlowParameters.create(parameters)
+                .setSlackBusSelectionMode(SlackBusSelectionMode.NAME)
+                .setSlackBusesIds(List.of("VL2_0"));
+
+        LoadFlow.run(network, parameters);
+
+        assertActivePowerEquals(401.757, network.getLine("L1").getTerminal1());
+        assertActivePowerEquals(100.878, network.getLine("L2").getTerminal1());
+        assertActivePowerEquals(100.878, network.getLine("L3").getTerminal1());
+
+        AcLoadFlowParameters acLoadFlowParameters
+                = OpenLoadFlowParameters.createAcParameters(network,
+                                                            parameters, olfParameters,
+                                                            new DenseMatrixFactory(),
+                                                            new NaiveGraphConnectivityFactory<>(LfElement::getNum),
+                                                            true,
+                                                            false);
+        try (LfNetworkList lfNetworks = Networks.load(network, acLoadFlowParameters.getNetworkParameters(), Collections.emptySet(), Set.of(c1), Reporter.NO_OP)) {
+            LfNetwork largestNetwork = lfNetworks.getLargest().orElseThrow();
+            largestNetwork.getBranchById("C1").setDisabled(true);
+            try (AcLoadFlowContext context = new AcLoadFlowContext(largestNetwork, acLoadFlowParameters)) {
+                new AcloadFlowEngine(context).run();
+            }
+            // should be the same as with previous LF
+            assertEquals(401.740, largestNetwork.getBranchById("L1").getP1().eval() * PerUnit.SB, LoadFlowAssert.DELTA_POWER);
```

Comment:
@annetill why we don't get exactly the value we had with the initial LF: 401.757 vs 401.740 ?

Reply:
Maybe because in the first case, we have a system resolution based on the bus/view and in the second case, we are based on the bus/breaker view.

Reply:
Yes right, this is not exactly the same system, so make sense to not have numerically the same solution